### DISCUSSION TITLE
qTree 'v-model' directives require no argument Error

### DISCUSSION THREAD

**Discussion by istvan0304** [quasarframework/quasar#11405] [General - Components / Directives / etc]
Hi, I'd use the qtree component and if I try to set expanded element I get an error:  'v-model' directives require no argument
How I solve this problem? According to the documentation I use like this:  v-model:expanded="expanded". If I leave "v-model" there is no error but don't work element expand when I click on it.

My code:
`<q-tree
            :nodes="props"
            node-key="label"
            v-model:expanded="expanded"
          />`

Data:
`data() {
expanded: [68],
}`

**Comment by rstoenescu** [comment]
This seems to work correctly. https://quasar.dev/vue-components/tree#example--no-connectors
Post a codepen so we can further help you.

**Reply by istvan0304** [reply]
Yeah, I know that works in the documentation :) And it's works in codepean too. Maybe something wrong with my environment. My vue version is 2.6
quasar version: v2.1.14
My IDE underline with red this section: v-model:expanded="expanded" 
and get error 
![kép](https://user-images.githubusercontent.com/19621571/142598681-42a8b97c-cd43-46d2-8574-b55c4c1f7489.png)

Any idea? Thanks.

**Comment by MilosPaunovic** [comment]
Quasar 2.x.x is built using Vue 3.x.x. That is the first issue to tackle in your environment.

**Comment by hawkeye64** [comment]
Maybe in your actual code, it's different, but the example for `data ()` above is incorrect.
It should be:
```js
data() {
  return {
    expanded: [68]
  }
}
```

You are missing the `return` of your data for the v-model.
